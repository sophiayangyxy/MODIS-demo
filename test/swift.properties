site=local
use.provider.staging=true
execution.retries=2

site.local {
  jobmanager=local
  initialScore=10000
  filesystem=local
  workdir=/tmp/swiftwork
}

app.local.getlanduse=/Users/yangyangxinye/MODIS-demo/test/bin/getlanduse
app.local.analyzelanduse=/Users/yangyangxinye/MODIS-demo/test/bin/analyzelanduse
app.local.markmap=/Users/yangyangxinye/MODIS-demo/test/bin/markmap
app.local.colormodis=/Users/yangyangxinye/MODIS-demo/test/bin/colormodis
app.local.ls=/bin/ls
