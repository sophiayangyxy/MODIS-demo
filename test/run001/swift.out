SwiftScript program does not exist: modis.swift

For usage information:  swift -help

